src/main/java/org/craftercms/studio/impl/v2/repository/GitContentRepository.java

```diff
@@ -512,25 +511,26 @@ public List<RepoOperation> getOperationsFromDelta(String site, String commitIdFr
     @Override
     public String getRepoFirstCommitId(final String site) {
         String toReturn = EMPTY;
+        String gitLockKey = StringUtils.isEmpty(site) ?
```

Comment:
```
StringUtils.isEmpty(site) ?
                                GLOBAL_REPOSITORY_GIT_LOCK :
                                helper.getSandboxRepoLockKey(site);
```

Seems to be repeated everywhere. Can we just change `getSandboxRepoLockKey` to receive an additional parameter, a boolean `ifSiteEmptyUseGlobalRepoLockKey`, and add this check in there?

Reply:
Fixed. Thank you!